config/jobs/kubernetes/sig-testing/verify.yaml

```diff
@@ -36,7 +38,7 @@ presubmits:
           privileged: true
         resources:
           requests:
-            cpu: 4
+            cpu: 6
```

Comment:
out of curiosity: do you know what parts of verify use more than a single core? I though that most of these are actually single-threaded...